src/overlays/actors/ovl_En_Suttari/z_en_suttari.c

```diff
@@ -21,110 +58,1507 @@ const ActorInit En_Suttari_InitVars = {
+            break;
+        case 0x29EC:
+            this->animationIdx = 7;
+            func_800BDC5C(&this->skelAnime, sAnimations, this->animationIdx);
+            break;
+        case 0x29ED:
+            this->animationIdx = 1;
+            func_800BDC5C(&this->skelAnime, sAnimations, this->animationIdx);
+            break;
+    }
+}
+
+void func_80BAAB78(EnSuttari* this, GlobalContext* globalCtx) {
+    if (this->flags1 & 4) {
+        switch (this->textId) {
+            case 0:
+                this->flags1 |= 0x20;
+                this->cutsceneIdx = 0;
+                this->textId = 0x29E5;
+                break;
+            case 0x29E5:
+                ActorCutscene_Stop(this->cutscenes[this->cutsceneIdx]);
+                this->cutsceneIdx = 1;
+                this->textId = 0x29E6;
+                break;
+            case 0x29E6:
+                ActorCutscene_Stop(this->cutscenes[this->cutsceneIdx]);
+                this->cutsceneIdx = 0;
+                this->textId = 0x29E7;
+                break;
+            case 0x29E7:
+                ActorCutscene_Stop(this->cutscenes[this->cutsceneIdx]);
+                this->cutsceneIdx = 1;
+                this->textId = 0x29E8;
+                break;
+            case 0x29E8:
+                ActorCutscene_Stop(this->cutscenes[this->cutsceneIdx]);
+                this->cutsceneIdx = 0;
+                this->textId = 0x29E9;
+                break;
+            case 0x29E9:
+                ActorCutscene_Stop(this->cutscenes[this->cutsceneIdx]);
+                this->cutsceneIdx = 1;
+                this->textId = 0x29EA;
+                break;
+            case 0x29EA:
+                ActorCutscene_Stop(this->cutscenes[this->cutsceneIdx]);
+                this->cutsceneIdx = 0;
+                this->textId = 0x29EB;
+                break;
+            case 0x29EB:
+                ActorCutscene_Stop(this->cutscenes[this->cutsceneIdx]);
+                this->cutsceneIdx = 1;
+                this->textId = 0x29EC;
+                break;
+            case 0x29EC:
+                ActorCutscene_Stop(this->cutscenes[this->cutsceneIdx]);
+                this->cutsceneIdx = 0;
+                this->textId = 0x29ED;
+                break;
+            case 0x29ED:
+                ActorCutscene_Stop(this->cutscenes[this->cutsceneIdx]);
+                this->flags1 |= 0x400;
+                this->cutsceneIdx = 1;
+                this->textId = 0x29EE;
+                break;
+        }
+        if (ActorCutscene_GetCurrentIndex() == 0x7C) {
+            ActorCutscene_Stop(0x7C);
+        }
+        ActorCutscene_SetIntentToPlay(this->cutscenes[this->cutsceneIdx]);
+        this->actionFunc = func_80BADD0C;
+    } else if (this->flags1 & 1) {
+        switch (this->textId) {
+            case 0:
+                if (gSaveContext.weekEventReg[0x51] & 1) {
+                    this->textId = 0x1455;
+                    ((EnElf*)PLAYER->naviActor)->unk264 |= 8;
```

Comment:
I don't know if there has been a decision reached if we are calling it navi or not. I do know there was discussion to try and have some kind of consistency between MM and OoT if possible.

Reply:
We are definitely not calling it Navi.
I'm good with any of these options...
`fairyTextId` and `fairyActor`
`tatlTextId` and `tatlActor`
`elfTextId` and `elfActor`
`companionTextId` and `companionActor`

Reply:
`this->textId` is actually Sakon's text, so will leave them as is. For `naviActor`, I like `companionActor` the best so will change it to that.

Reply:
Please change both while you are in there.

Reply:
Ah, you were talking about the player struct. Yeah, I can do that.

Reply:
Looks like the overwhelming consensus on Discord was to just use `tatl`, so will do that.